deploy-manage/security/secure-cluster-communications.md

```diff
@@ -1,93 +1,134 @@
+Both {{es}} and {{kib}}, the core components of the {{stack}}, expose service endpoints that must be secured. {{es}} handles traffic at two levels:
+* The **transport layer**, used for internal communication between nodes in the cluster.
+* The **HTTP layer**, used by external clients — including Kibana — to send requests via the REST API.
 
-| **Channel** | **Description** | **Security needs** |
-|-------------|-----------------|--------------------|
-| [Transport layer](#transport-layer-security) | Communication between {{es}} nodes within a cluster | - Mutual TLS (required)<br>- Node authentication<br>- Node role verification |
-| [HTTP layer](#http-layer-security) | Communication between external clients and {{es}} through the REST API | - TLS encryption<br>- Authentication (basic auth, API keys, or token-based)<br>- Optional client certificate authentication |
-| [{{kib}}-to-{{es}}](#kib-to-es-communications) | Communication from the {{kib}} server to {{es}} for user requests and queries | - TLS encryption<br>- Service authentication (API keys, service tokens, or mutual TLS) |
+Additionally, {{kib}} functions as a web server, exposing its own **HTTP endpoint** to users, and also acts as a client when sending requests to {{es}}.
+
+To ensure secure operation, it’s important to understand the communication channels and their specific security requirements.
 
+| **Channel** | **Description** | **TLS requirements** |
+|-------------|-----------------|--------------------|
+| [{{es}} transport layer](#transport-layer-security) | Communication between {{es}} nodes within a cluster | Mutual TLS/SSL required for multi-node clusters |
+| [{{es}} HTTP layer](#http-layer-security) | Communication between external clients and {{es}} through the REST API | TLS/SSL optional (but recommended) |
+| [{{kib}} HTTP layer](#http-layer-security) | Communication between external browsers and {{kib}} through the REST API | TLS/SSL optional (but recommended) |
 
-## Transport layer security
+### Transport layer security
 
-The transport layer is used for communication between {{es}} nodes in a cluster. Securing this layer prevents unauthorized nodes from joining your cluster and protects internode data.
+The transport layer is responsible for internal communication between {{es}} nodes in the cluster. Securing this layer prevents unauthorized nodes from joining your cluster and protects internode traffic.
 
 The way that transport layer security is managed depends on your deployment type:
 
-- **ECH, ECE, and Serverless**: Transport security is fully managed by Elastic. No configuration is required.
-- **ECK**: Transport security is automatically configured by the operator, but you can [customize its service and SSL certificates](/deploy-manage/security/k8s-transport-settings.md).
-- **Self-managed**: Transport security must be manually configured following the steps in [Set up basic security](set-up-basic-security.md).
+::::{tab-set}
```

Comment:
I think i'd prefer these to stay as bullets because they're just one line. 

generally would suggest only using tabs for parallel procedures

Reply:
I see the tabs down below are a little bigger. ok with keeping them for parallelism and see if we like them. HOWEVER, you should [sync the tabs](https://elastic.github.io/docs-builder/syntax/tabs/#tab-groups) so when someone elects one, it switches the tabs for all of the sets.